CHANGELOG.md

```diff
@@ -9,9 +9,34 @@ This changelog attempts to conform to the changelog spec on [keepachangelog.org]
 The evergreen, canonical changelog for *all NILRT branches* can be [found here](https://github.com/ni/nilrt/blob/HEAD/CHANGELOG.md).
 
 ## Latest Updates
-* nilrt: 301
-* meta-nilrt: 812
-* linux: 215
+* nilrt: 307
+* meta-nilrt: 852
+* linux: 235
+
+
+----
+## 11.2
+Branch: `nilrt/25.5/scarthgap`
+
+### nilrt
+No changes.
+
+### meta-nilrt
+
+
+#### Added
+- [Added](https://github.com/ni/meta-nilrt/pull/835) support for wired `802.1X`.
+- [Added](https://github.com/ni/meta-nilrt/pull/839) `nilrt-logging` to install INI file to specify log paths.
+- [Added](https://github.com/ni/meta-nilrt/pull/840) persistent logging.
```

Comment:
Persistent logging support was already there - we just changed the setting so it's enabled by default.

Better to move this under a `### Changed` section and reword to `[Enabled](https://github.com/ni/meta-nilrt/pull/840) persistent logging by default`.